apple/Sources/Sargon/Extensions/Swiftified/System/BIOS/BIOS+Swiftified.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  File.swift
+//  
+//
+//  Created by Alexander Cyon on 2024-05-03.
+//
+
+import Foundation
+import SargonUniFFI
+
+public typealias BIOS = Bios
+extension BIOS: @unchecked Sendable {}
+
+
+extension Drivers {
+	public convenience init(
+		bundle: Bundle,
+		userDefaultsSuite: String,
+		secureStorageDriver: SecureStorageDriver
+	) {
+		self.init(
+			appVersion: (bundle.infoDictionary?["CFBundleShortVersionString"] as? String) ?? "Unknown",
```

Comment:
> Any reason for having this logic injected to `HostInfo` instead of simply resolving inside such driver?

I think an upstream Swift package cannot access a downstream consuming app targets main Bundle?

Reply:
oh so this would get injected the host app `Bundle`?

Reply:
Wallet passes along the info to Swift Sargon is the idea yes